src/components/download-links/style.scss

```diff
@@ -1,7 +1,8 @@
 download-links {
   position: relative;
-  margin-left: auto;
+  margin-left: 0;
   display: flex;
+  border-left: 0 !important;
```

Comment:
I took from another component that had that, which Luke had written in; not sure what the use case was, but didn't want to break by deviating!